skapa-ett-konfetti-regn-main/confetti.js

```diff
@@ -0,0 +1,52 @@
+// Om du tittar på projektet i browsern ser du en ensam grå
+// och lite sorglig konfettipartikel. Din uppgift blir att
+// fylla konfettianimationen med färger och liv!
+
+// Funktionen chooseParticleColor styr färgen på varje enskild
+// partikel. Den ska returnera en text-sträng som innehåller en
+// CSS-färg. Returnerar den samma färg varje gång så kommer alla
+// partiklar ha samma färg men om den returnerar olika färger
+// t.ex. baserat på slump eller baserat på inparametern groupName
+// så kan det istället bli en blandning av färger.
+const chooseParticleColor = (groupName) => {
+  var letters = '0123456789ABCDEF';
+  var color = '#';
+  for (var i = 0; i < 6; i++) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
```

Comment:
Den här lösningen funkar absolut, den löser uppgiften och slumpar in alla möjliga färger som finns, snyggt!

Tips till den här typ av lösning är att man kan välja returnera RGB värden och jobba direkt med siffror, som i RGB(0, 0, 0); 

Till exempel
```
return `RGB(${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)})`
```

Eller för hex värden så har Nummer i javascript en toString metod där man kan skicka in 16 för att konvertera till hex. Så Number(255).toString(16) ger 'f'. Då kan man även returnera en del färgkod baserad på ett slumpat värde mellan 0 och 16777215 som är max antal olika färger man kan få från RGB 0, 0, 0 till RGB 255, 255, 255.

Det kan da se såhär ut
`return '#' + Math.floor(Math.random()*16777215).toString(16)`